<launch>
  <node type="nifti_laser_assembler" pkg="nifti_laser_assembler" name="nifti_laser_assembler"/>
</launch>
